hw/ip/spi_device/README.md

```diff
@@ -76,7 +76,7 @@ The IP supports Normal Read, Fast Read, Fast Read Dual Output, Fast Read Quad Ou
 This version of IP does not support Dual IO, Quad IO, QPI commands.
 
 In Passthrough mode, SPI Device receives SPI transactions from a host system and forwards the transactions to a downstream flash device.
-SW may filter prohibited commands by configuring 256-bit [`FILTER`](data/spi_device.hjson#filter) CSR.
+SW may filter prohibited commands by configuring 256-bit [`FILTER`](doc/registers.md#filter) CSR.
```

Comment:
This anchor does not exist. How about...
```suggestion
SW may filter prohibited commands by configuring the 256-bit FILTER CSR ([`CMD_FILTER_0`](doc/registers.md#cmd_filter_0) through ['CMD_FILTER_7'](doc/registers.md#cmd_filter_7)).
```

Reply:
For this PR, I've avoided making any non-trivial register link fixes. There are a few registers linked to that no longer exist. Fragment checking in CI is in the pipeline.